<launch>
    <node pkg="lane_detection" type="lane_detection_node" name="lane_detection_node">
        <!-- can save last img -->
        <param name="save_last_img" type="bool" value="false"/>

        <!-- get color by HSV -->
        <param name="hue_1_max" type="int" value="10"/>
        <param name="hue_1_min" type="int" value="0"/>
        <param name="hue_2_max" type="int" value="179"/>
        <param name="hue_2_min" type="int" value="159"/>
        <param name="sat_max" type="int" value="255"/>
        <param name="sat_min" type="int" value="40"/>
        <param name="val_max" type="int" value="255"/>
        <param name="val_min" type="int" value="120"/>

        <!-- show process -->
        <param name="show_source" type="bool" value="true"/>
        <param name="show_reduced" type="bool" value="false"/>
        <param name="show_sliding_window" type="bool" value="true"/>

        <!-- waypoint hegiht -->
        <param name="waypoint_height" type="int" value="3"/>
    </node>
</launch>
